Makefile

```diff
@@ -165,7 +165,7 @@ cache-era5-gcp: ensure-era5-cached
 	gcloud storage cp -r -n .doc-test-cache/* gs://contrails-301217-unit-test/doc-test-cache/
 
 doctest: ensure-era5-cached ensure-gcp-credentials
-	pytest --doctest-modules \
+	PYCONTRAILS_CACHE_DIR="$(CURDIR)/.doc-test-cache" pytest --doctest-modules \
```

Comment:
These ENV variables are set in the Github action (See for example [doctest action](https://github.com/contrailcirrus/pycontrails/blob/2c06549b985ee8af7158163ca908fd7af7a4997d/.github/workflows/doctest.yaml#L22)

I assume this will override the global ENV, but perhaps this is generic as to work everywhere? Just make sure you have that reconciled.